app/views/layouts/_header.html.erb

```diff
@@ -76,6 +85,15 @@
           <li><%= link_to t("layouts.copyright"), copyright_path, :class => "dropdown-item" %></li>
           <li><%= link_to t("layouts.help"), help_path, :class => "dropdown-item" %></li>
           <li><%= link_to t("layouts.about"), about_path, :class => "dropdown-item" %></li>
+          <li>
+            <% if current_user && current_user.id %>
+              <%= link_to(preferences_path) do %>
+                <%= render "shared/language_selector", :hoverable => true, :black => true, :classes => "dropdown-item", :disabled => true %>
```

Comment:
`dropdown-item` here doesn't entirely work, and possibly other Bootstrap classes either don't work as expected or are missing. Some related odd behavior of language link/label:

- not logged in, no compact mode: no focus outline
- not logged in, compact mode: same + keyboard up/down controls stop working correctly
- logged in, no compact mode: focus outline is present but non-standard because no `nav-link`, but if you add it you'll need more css fixes
- logged in, compact mode: keyboard controls also not working correctly, in a slightly different manner

Reply:
I'll try to solve the focusing issue without any custom classes. In terms of collapsibility of the translation icon, purpose was to take as little space as possible when the screen is smaller. Without collapsing, HTML code will be shorter and with less "if-else"-es.

Reply:
Do you still want this icon collapsing into *More*?

Reply:
I'll try to move it out of the "More" and test with different resolutions. If it doesn't break any resolution, I'll keep it out of the "More". It will need less HTML code.

Reply:
I've updated this PR. Tried several solutions and current one needed the smallest and cleanest code. Also, one CSS class was added `.text-transparent` to make text and background of the `<select>` tag visually hidden but still focusable.

Language selector will no more be moved to the compact menu. Visually it doesn't take much space and contextually it's more relevant to be as an icon than as a compacted menu button.